modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -53,25 +54,52 @@ data:
     prometheusK8s:
       remoteWrite:
       - url: "https://remote-write.endpoint"
-        <endpoint_authentication_credentials>
+        <endpoint_authentication_credentials> <1>
 ----
 +
-For `endpoint_authentication_credentials` substitute the credentials for the endpoint.
-Currently supported authentication methods are basic authentication (`basicAuth`) and client TLS (`tlsConfig`) authentication.
+<1> For `endpoint_authentication_credentials` substitute the authentication method and the credentials for the endpoint.
+Currently supported authentication methods are AWS Signature version 4 (`sigv4`), basic authentication (`basicAuth`),  bearer token (`Bearer`), TLS client (`tlsConfig`), and OAuth 2.0 (`oauth2`). 
++
+* AWS Signature Version 4
++
+This method uses AWS Signature Version 4 (`sigv4`) authentication to sign requests.
+This method cannot be used simultaneously with `basic_auth`, `Bearer`, or `oauth2`.
 +
-* The following example configures basic authentication:
+The following shows a `sigv4` sample configuration.
+It assumes you have already created a `sigv4` secret named `sigv4-credentials`in the `openshift-monitoring` namespace.
```

Comment:
`sigv4-credentials`in 
should be
`sigv4-credentials` in

Reply:
fixed